lib/engine/game/base.rb

```diff
@@ -618,6 +618,7 @@ def process_action(action)
       rescue Engine::GameError => e
         @exception = e
         @actions |= [action]
```

Comment:
I'm not sure if this line should be deleted or not--if the action is broken, should we prevent it from being added to `@actions`?

Reply:
deleted